src/message_interface/message_handler.rs

```diff
@@ -230,6 +230,28 @@ impl WalletMessageHandler {
                 })
                 .await
             }
+            Message::GenerateAddress {
+                account_index,
+                internal,
+                address_index,
+                options,
+            } => {
+                convert_async_panics(|| async {
+                    let address = self
+                        .account_manager
+                        .generate_address(account_index, internal, address_index, options)
+                        .await?;
+
+                    let bech32_hrp = self
+                        .account_manager
+                        .get_bech32_hrp()
+                        .await
+                        .unwrap_or_else(|_| SHIMMER_TESTNET_BECH32_HRP.to_string());
```

Comment:
Should we really be silent about this ?

Reply:
I think to be able to generate the addresses offline without an account it should be like this, unless we let the user provide the bech32 hrp somewhere

Reply:
Yes, because it will otherwise not work in an offline environment?

Reply:
Can remove it now ?